setup.cfg

```diff
@@ -0,0 +1,46 @@
+[metadata]
+name = topostats
+version = 0.1.0
+description = Automated Analysis for Atomic Force Microscopy Images of Biomolecules
+long_description = file: README.md
+url = https://github.com/AFM-SPM/TopoStats
+author = TopoStats Team
+license = GNU Lesser GPLv3
+
+[options]
+python_requires = >=3.8
```

Comment:
It no doubt has an effect there (e.g. if the test matrix included Python 3.7 it would fail), but this is really in place for when users are installing to ensure they have Python `>= 3.8`.